default.nix

```
@@ -750,6 +751,11 @@ rec {
     builtins.attrValues js;
   };
 
+  viperServer = builtins.fetchurl {
```

Do this to get rid of the eval error on hydra:
```suggestion
  viperServer = nixpkgs.fetchurl {
```